src/PrestaShopBundle/Resources/config/services/core/product.yml

```diff
@@ -11,4 +11,7 @@ services:
     class: PrestaShop\PrestaShop\Core\Product\Combination\Generator\CombinationGenerator
 
   prestashop.core.product.combination.name_builder.combination_name_builder:
+    arguments:
+      - '@translator'
+      - ', '
```

Comment:
A little but surprising if you did not read the constructor signature before 😝

Reply:
we can inject in the future a php const or an env variable instead of this string. But for now this setup is done outside the service, and it's a first step.

Reply:
Or even a configuration from DB, but it can be surprising at first sight indeed ^^ Maybe we should add a comment on this one

Reply:
I added a comment to explain what is it, and what's next